modules/programs/aerospace.nix

```diff
@@ -0,0 +1,231 @@
+{ config, lib, pkgs, ... }:
+let
+  inherit (lib) mkOption types;
+  cfg = config.programs.aerospace;
+
+  tomlFormat = pkgs.formats.toml { };
+
+  filterAttrsRecursive = pred: set:
+    lib.listToAttrs (lib.concatMap (name:
+      let v = set.${name};
+      in if pred v then
+        [
+          (lib.nameValuePair name (if lib.isAttrs v then
+            filterAttrsRecursive pred v
+          else if lib.isList v then
+            (map (i: if lib.isAttrs i then filterAttrsRecursive pred i else i)
+              (lib.filter pred v))
+          else
+            v))
+        ]
+      else
+        [ ]) (lib.attrNames set));
+  filterNulls = filterAttrsRecursive (v: v != null);
+in {
+  meta.maintainers = with lib.hm.maintainers; [ damidoug ];
+
+  options.programs.aerospace = {
+    enable = lib.mkEnableOption "Whether to enable AeroSpace window manager.";
```

Comment:
Please check the generated documentation. This produces

> Whether to enable Whether to enable AeroSpace window manager…

See <https://nix-community.github.io/home-manager/options.xhtml#opt-programs.aerospace.enable>.